vaadinfrontend/frontend/themes/datamanager/components/main.css

```diff
@@ -0,0 +1,25 @@
+.experiment-main {
+  display: grid;
+  grid-template-columns: minmax(250px, 0.8fr) minmax(250px, 0.2fr);
+  grid-template-rows: minmax(150px, 0.1fr) minmax(250px, 1fr);
+  gap: 1px 1px;
+  grid-template-areas:
+    "navbar support"
+    "content support";
+}
+
+.experiment-main .content {
+  grid-area: content;
+}
+
+.experiment-main .support {
+  grid-area: support;
+}
+
+.experiment-main .navbar {
+  grid-area: navbar;
+}
+
+.project-main {
+  flex-direction: column;
+}
```

Comment:
why not extract common `main` styling into the `.main` class and use `.main.experiment` and `.main.project` instead of different classes? Is there common styling?

Reply:
That's a valid point i'll adapt the styling to be more general! 👍